### DISCUSSION TITLE
alternate links

### DISCUSSION THREAD

**Discussion by Potato-constellation** [UseInterstellar/Interstellar#640] [General]
hello i was wondering if anyone else has some alternate links for interstellar (also im sorry if i made this wrong im new to github)

**Comment by replitidk** [comment]
@Potato-constellation  if you want links go to the discord we dont share links in github or anything like that

**Comment by xbubbo** [comment]
Join our [discord](https://discord.gg/interstellar) or make your own links [here](https://github.com/UseInterstellar/Interstellar?tab=readme-ov-file#deployment).
